conformance/ion_encoding/module/macro_table.ion

```diff
@@ -0,0 +1,36 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+// Trivial macro_table in inline module; no macro definitions
+
+
+(ion_1_1
+  (each (toplevel $ion_encoding::((module M (macro_table))))
+        (encoding (module M (macro_table)))
+        (mactab)
+        (produces)))
+
+(ion_1_1
+  (encoding (module M (macro_table))
+            (macro_table M))
+  (toplevel ('#$:M:mac'))
+  (signals "No such macro: :M:mac"))
+
+(ion_1_1 "Pass-through of visible module mactab"
+  (encoding (module M (macro_table $ion))
+            (macro_table M))
+  (each (toplevel ('#$:M:make_symbol' "sah" weet))
+        (toplevel ('#$:M:3'           "sah" weet))
+        (produces sahweet)))
+
+
+// Macro addresses
+
+(ion_1_1
+  (encoding (module M (macro_table $ion))
+            (macro_table M))
+  (toplevel
+//              ('#$:make_symbol' ('#$:values' "a" b) ('#$:void' x) c)  // TODO [MAC-ID]
```

Comment:
What does "TODO [MAC-ID]" mean?

Reply:
My model doesn't yet implement macro identity, so it can't distinguish when a name at two addresses points to the same macro.   I'm hoping to implement that this week, so I'll hold off on merging this until that works.

Reply:
Pushed a new commit that enables this; I've implemented disambiguation in the model.